substrafl/dependency/schemas.py

```diff
@@ -22,17 +23,31 @@ class Dependency(BaseModel):
             If set to True, it will be the one installed in editable mode from your python environment.
             Defaults to False.
         dependencies (List[str]): Python packages installable from pypi.
-        local_dependencies (List[pathlib.Path]): Local installable packages. The command
+        local_dependencies (List[pathlib.PosixPath]): Local installable packages. The command
```

Comment:
Why change to posix ?

Reply:
I think we want to stay with Path to support Windows path

Reply:
The docstring does not match the definition. All these values are _already_ defined as `PosixPath`.

Reply:
Its define at PosixPath because we are on linux but would be windows path if we were on windows no ?

Reply:
https://docs.python.org/3/library/pathlib.html